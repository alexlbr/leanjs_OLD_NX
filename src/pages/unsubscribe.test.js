import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Unsubscribe from './unsubscribe'

describe('<Unsubscribe />', () => {
  it('should render component', () => {
    const wrapper = shallow(<Unsubscribe />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
