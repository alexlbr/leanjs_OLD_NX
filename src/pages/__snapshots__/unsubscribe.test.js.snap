// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Unsubscribe /> should render component 1`] = `
<Fragment>
  <Header
    bgImg="about-us"
    subtitle="You are now unsubscribed from recieving further emails, for any further support please contact us here: unsubscribe@reactjs.academy"
    titleLines={
      Array [
        "Unsubscribe from email communications",
      ]
    }
  />
  <TopSection>
    <Grid>
      <Card
        border="shadow"
      >
        <Row>
          <Styled(Col)
            md={10}
            mdOffset={1}
          >
            <styled.h2>
              We are sorry to see you go! If you'd like to re-subscribe to our maining list at any time you can do so by re-entering your email address or contact us directly: hello@reactjs.academy
            </styled.h2>
          </Styled(Col)>
        </Row>
      </Card>
    </Grid>
  </TopSection>
</Fragment>
`;
